pkg/apis/pipeline/v1beta1/pipelinerun_validation.go

```diff
@@ -44,6 +45,18 @@ func (ps *PipelineRunSpec) Validate(ctx context.Context) (errs *apis.FieldError)
 		errs = errs.Also(apis.ErrMissingField("pipelineref.name", "pipelinespec"))
 	}
 
+	// Check that if a bundle is specified, that a PipelineRef is specified as well.
+	if (ps.PipelineRef != nil && ps.PipelineRef.Bundle != "") && ps.PipelineRef.Name == "" {
+		errs = errs.Also(apis.ErrMissingField("pipelineref.name"))
+	}
```

Comment:
This case isn't covered by a test is it ? :stuck_out_tongue_closed_eyes:

Reply:
Yeah I had realized that I couldn't test this because we can't hit this case. We are checking above it that either a spec or name is set so I might as well just delete this check since it can't happen with the code as is.